src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/DraftApprenticeship.cs

```diff
@@ -347,5 +347,131 @@ private List<DomainError> ValidateDraftApprenticeshipRplExtendedDetails(int? dur
 
             return errors;
         }
+
+        public void SetPriorLearningData(int? trainingTotalHours, int? durationReducedByHours, bool? isDurationReducedByRpl, int? durationReducedBy, int? costBeforeRpl, int? priceReduced)
+        {
+
+            if (RecognisePriorLearning != true)
```

Comment:
Could be:

`if (!RecognisePriorLearning)`

Reply:
again it's a bool? so we can;t